### PR TITLE
Dedupe Content Repo

### PR DESCRIPTION
**Pull request by avidan-H**
## Status
Ready

## Description
Deduplicates the content repo in preparation for `content_creator.py`/`demisto-sdk` processing packs. Removed all of the original content items that were copied into the CortexXDR pack.

## Required version of Demisto
any

## Does it break backward compatibility?
   - No

## Must have
- [ ] Tests
- [ ] Code Review

### CODE REVIEW

**Comment by glicht** [comment]
Does the PR support using the content from the packs dir?

**Comment by yardensade** [comment]
@liorblob please take a look at this as well

File: .circleci/config.yml [hunk 1]
```diff
@@ -68,8 +68,8 @@ jobs:
             python -m pytest ./Tests/scripts/infrastructure_tests/validate_files_test.py -v
 
             python -m pytest ./Tests/scripts/test_configure_tests.py -v
-            python3 -m pytest ./Tests/scripts/infrastructure_tests/update_id_set_test.py -v
-            python3 -m pytest ./Tests/scripts/infrastructure_tests/secrets_test.py -v
+            # No need to run 'update_id_set_test.py' and 'secrets_test.py' in 'Tests/scripts/infrastructure_tests'
```
Comment (yardensade) [inline .circleci/config.yml]: No need in these comments, this should be documented somewhere if anything.

File: .hooks/pre-commit [hunk 2]
```diff
@@ -15,14 +15,10 @@ if [ -e "./venv/bin/activate" -a -z "${VIRTUAL_ENV}" ]; then
...
-if [[ -z "${WINDIR}" ]]
-    then
-        PYTHONPATH="`pwd`:${PYTHONPATH}" python Tests/scripts/validate_files.py -t true
-        PYTHONPATH="`pwd`:${PYTHONPATH}" python Tests/scripts/hook_validations/secrets.py
-    else
-        python Tests/scripts/validate_files.py
-        python Tests/scripts/hook_validations/secrets.py
-fi
+
+python3 -m pip install demisto-sdk
```
Comment (yardensade) [inline .hooks/pre-commit]: Why are we pip installing this? shouldn't it be part of the bootstrap?

**Comment by liorblob** [comment]
Didn't understand the changes in `Tests/scripts`, are we deleting those files?

**Comment by avidan-H** [comment]
> Didn't understand the changes in `Tests/scripts`, are we deleting those files?

... Those changes are from your branch

**Comment by avidan-H** [comment]
@glicht I checked an instance from one of the recent builds and the content items from the pack are present.

**Comment by Itay4** [comment]
@glicht can you review again please ?

**Pull request merged**